packages/devtools/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.1.9 - 2019-10-09
```

Comment:
Does this change need to be published right away? we have a publish planned for (hopefully) later this week, so is it fine if we wait until then?